concepticondata/conceptlists/Smeltzer-1995-300.tsv

```diff
@@ -0,0 +1,301 @@
+ID	NUMBER	FRENCH	CONCEPTICON_ID	CONCEPTICON_GLOSS	ENGLISH
+Smeltzer-1995-300-1	1	oeil	1248	EYE	eye
+Smeltzer-1995-300-2	2	tête	1256	HEAD	head
+Smeltzer-1995-300-3	3	dent	1380	TOOTH	tooth
+Smeltzer-1995-300-4	4	langue	1205	TONGUE	tongue
+Smeltzer-1995-300-5	5	nez	1221	NOSE	nose
+Smeltzer-1995-300-6	6	oreille	1247	EAR	ear
+Smeltzer-1995-300-7	7	cou	1333	NECK	neck
+Smeltzer-1995-300-8	8	sein	1402	BREAST	breast
+Smeltzer-1995-300-9	9	ongle	1258	FINGERNAIL	fingernail
+Smeltzer-1995-300-10	10	ventre	1251	BELLY	belly
+Smeltzer-1995-300-11	11	nombril	1838	NAVEL	navel
+Smeltzer-1995-300-12	12	genou	1371	KNEE	knee
+Smeltzer-1995-300-13	13	peau	763	SKIN	skin
+Smeltzer-1995-300-14	14	plume	1201	FEATHER	feather
+Smeltzer-1995-300-15	15	corne	1393	HORN (ANATOMY)	horn
+Smeltzer-1995-300-16	16	queue	1220	TAIL	tail
+Smeltzer-1995-300-17	17	oeuf	744	EGG	egg
+Smeltzer-1995-300-18	18	lait	635	MILK	milk
+Smeltzer-1995-300-19	19	homme	1554	MAN	man
+Smeltzer-1995-300-20	20	femme	962	WOMAN	woman
+Smeltzer-1995-300-21	21	pèr			father?
```

Comment:
I found beau-père [french] and  père [french]  in Key-2016. But I'd leave it unmapped if we cannot be sure that it is a typo.

Reply:
This is one of the cases where I suspect there is a typo in père ('father').